src/components/ButtonComponents/OperatorButtons/OperatorButton.js

```diff
@@ -1,9 +1,13 @@
 import React from "react";
 
-const OperatorButton = () => {
+import { StyledButton } from '../SpecialButtons/SpecialButton';
+
+const OperatorButton = props => {
   return (
-    <>
-      {/* Display a button element rendering the data being passed down from the parent container on props */}
-    </>
+    <StyledButton operator onClick={() => props.operation(props.operator)}>
```

Comment:
Step 2: instead of this, you would have something like this:
`<StyledButton operator onClick={() => props.operation(props.operator.value)}>
      {props.operator.char}
    </StyledButton>`